lib/bitfield.h

```diff
@@ -70,7 +72,7 @@ typedef struct {word_t *data; size_t n, m; } bitfield_t;
 	do {                                                                   \
 		(v).n = 0;                                                     \
 		(v).m = ((N) / WORD_SIZE + 1);                                 \
-		(v).data = calloc(1, ((v).m * sizeof(word_t)));                \
+		(v).data = XCALLOC(MTYPE_BITFIELD, ((v).m * sizeof(word_t)));  \
```

Comment:
Shouldn't we make the corresponding change in `bf_free` to use XFREE?

Reply:
Yes of course. Doh!